vendor.conf

```diff
@@ -57,7 +57,7 @@ github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc374
 github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
 github.com/coreos/etcd v3.2.1
 github.com/coreos/go-semver v0.2.0
-github.com/ugorji/go f1f1a805ed361a0e078bb537e4ea78cd37dcf065
+github.com/ugorji/go go111 https://github.com/kolyshkin/ugorji-go
```

Comment:
Can you add a comment referring to the PR we're waiting on to be merged upstream? (Or a comment explaining why we're using a fork here)

Reply:
git blame shows it clearly; anyway, added :thumbsup: